BookClub/Models/Book.cs

```diff
@@ -2,15 +2,34 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
 
 namespace DevDo.BookClub.Models
 {
     public class Book
     {
-       
+        [Required(ErrorMessage = "El campo título es requerido.")]
+        [Display(Name = "Titulo")]
         public string Title { get; set; }
+
+        [Required(ErrorMessage = "El campo autor es requerido.")]
+        [Display(Name = "Autor")]
         public string Author { get; set; }
+
+        [StringLength(13)]
+        [Required(ErrorMessage = "El campo ISBN es requerido.")]
         public string ISBN { get; set; }
+
+        [Required(ErrorMessage = "La imágen es requerida..")]
+        [Display(Name = "Imágen")]
         public string CoverUrl { get; set; }
+
+        public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+        {
+            if (ISBN.Length > 13)
```

Comment:
Usually, code that is not being used, even if it might be used in the future does not go in. More often than not, you never actually use that code!. Future you will appreciate the fact that he doesn't have to think about what that code does because it doesn't exist. Worst case scenario you get a new feature/issue that makes you introduce it.